flink-java/src/main/java/org/apache/flink/api/java/functions/FormattingMapper.java

```diff
@@ -23,6 +23,10 @@
 import org.apache.flink.api.java.io.TextOutputFormat;
 import org.apache.flink.api.java.io.TextOutputFormat.TextFormatter;
 
+/**
+ * Mapper that formats value into {@link String} representation.
```

Comment:
-> "Mapper that converts values to strings using a {@link TextFormatter}."

I don't think we need to link to the String class ;)